dev-libs/zziplib/zziplib-0.13.71_p20200419.ebuild

```diff
@@ -35,26 +32,39 @@ RDEPEND="${DEPEND}"
 S="${WORKDIR}/${PN}-${MY_COMMIT}"
 
 PATCHES=(
-	"${FILESDIR}"/${PN}-0.13.69-009-perror.patch
-	"${FILESDIR}"/${PN}-0.13.71-installing-man3-pages.patch
+	"${FILESDIR}"/${PN}-0.13.69-009-perror.patch # not clear why this patch is needed
```

Comment:
This is not really convincing, please study the patch and update why it's needed. Or remove this line/patch if it's not needed.

Reply:
removed